experimental/null_safety/README.md

```diff
@@ -0,0 +1,97 @@
+# Null safety Flutter example
+
+This folder contains a demonstration of the upcoming Dart feature, sound null
+safety. For additional information about this feature, see
+https://dart.dev/null-safety.
+
+The folder contains two sample Flutter apps:
+
+  * `null_unsafe_app`: This contains a small weather app, which has three null
+    errors in its rendering code as a result of not using null safety.
+    
+    Try running the app (`flutter run` in a terminal, or the Run action in
+    VSCode/Android Studio), and see if the UI works. The null errors are
+    sporadic, so try hot reloading a bunch of times and experience the app
+    sometimes working and sometimes failing.
+
+  * `null_safe_app`: This contains the same weather app, but this time using
+    null safety. There are two copies of the source code:
+
+    `lib/badapp.dart`: This is the same source code as in the unsafe app. Try
+    uncommenting the two commented lines of null-unsafe code, and notice how the
+    Dart analyzer with null safety immediately flags the lines as unsafe.
+
+    `lib/goodapp.dart`: This a safe version of the unsafe app, with the null
+    errors fixed. Read through the comments to see illustrations of how to resolve
+    null errors with null safety.
+
+## Running the example code
+
+Null safety is still in technical preview. As a result you need to use a
+dev-channel Flutter build, and you have to pass an [experiment
+flag](https://dart.dev/tools/experiment-flags).
+
+### Flutter preview SDK installation
+
+  1. Download the latest null safety preview build from the **dev channel** of
+     the Flutter SDK releases page:
+     https://flutter.dev/docs/development/tools/sdk/releases.
+        
+  1. Unzip the SDK to a folder, e.g. `/Users/michael/dev/preview/flutter` (on
+     macOS/Linux) or `C:\Users\michael\dev\preview\flutter\` (on Windows).
+
+### Running from the terminal/command-prompt
+
+Because null safety is still in tech preview, we need to pass a so-called
+_experiment flag_ when invoking any Flutter command in the terminal, which looks
+like this: `--enable-experiment=non-nullable`.
+
+To run the main app, type these commands in the terminal/command-prompt:
+
+  - Windows:
+    - `cd <folder with samples repo>\null_safety\calculate_lix\`
```

Comment:
Is `calculate_lix` left over from an earlier version?

Reply:
Good catch, fixed